docs/guides/getting-started/setup/html-css-js.mdx

```diff
@@ -1,14 +1,14 @@
 ---
-description: Build a Tauri app from scratch using just HTML, CSS, and JavaScript
+description: Build a Tauri app using HTML, CSS, JavaScript using either Node or Cargo
 pagination_next: guides/architecture/README
 sidebar_custom_props:
   doc_card_image: /img/guides/getting-started/setup/html5-light.svg
   doc_card_image_dark: /img/guides/getting-started/setup/html5-dark.svg
 ---
 
-import TauriInit from './_tauri-init.mdx'
-import Commands from './_commands.mdx'
-import Intro from './_intro.mdx'
+import TauriInit from './_fragments/_tauri-init.mdx'
+import Commands from './_fragments/_commands.mdx'
+import Intro from './_fragments/_intro.mdx'
 import Command from '@theme/Command'
 
 # HTML/CSS/JavaScript
```

Comment:
Slightly off-topic: We should rename this to something like`HTML, CSS, and JavaScript` as the current name gets treated as one word by css word wrapping. This causes the name to go off-screen on mobile

Reply:
Just updated this and pushed